packages/framework-core/src/services/token-verifiers/utilities.ts

```diff
@@ -0,0 +1,88 @@
+import { DecodedToken } from '@boostercloud/framework-types'
+import * as jwt from 'jsonwebtoken'
+import { JwksClient, SigningKey } from 'jwks-rsa'
+
+/**
+ * Initializes a jwksRSA client that can be used to get the public key of a JWKS URI using the
+ * `getKeyWithClient` function.
+ *
+ * @param jwksUri The JWKS URI
+ * @returns A JwksRSA client
+ */
+export function getJwksClient(jwksUri: string): JwksClient {
+  const jwksRSA = require('jwks-rsa') // Manually loading the default export here to be able to stub it
+  return jwksRSA({
+    jwksUri,
+    cache: true,
+    cacheMaxAge: 15 * 60 * 1000, // 15 Minutes, at least to be equal to AWS max lambda limit runtime
+  })
+}
+
+/**
+ * Initializes a function that can be used to get the public key from a JWKS URI with the signature
+ * required by the `verifyJWT` function. You can create a client using the `getJwksClient` function.
+ *
+ * @param client A JwksClient instance
+ * @param header The JWT header
+ * @param callback The callback function that will be called when the public key is ready
+ * @returns A function that can be used to get the public key
+ */
+export function getKeyWithClient(client: JwksClient, header: jwt.JwtHeader, callback: jwt.SigningKeyCallback): void {
+  if (!header.kid) {
+    callback(new Error('JWT kid not found'))
+    return
+  }
+  client.getSigningKey(header.kid, function (err: Error | null, key: SigningKey) {
+    if (err) {
+      callback(err)
+      return
+    }
+    callback(null, key.getPublicKey())
+  })
+}
+
+/**
+ * Strips the `Bearer` prefix from the Authorization header so it can be fed to a TokenVerifier `verify` method.
+ * @param authorizationHeader The value of the Authorization header
+ * @returns The token part of the Authorization header
+ */
+export const sanitizeAuthorizationHeader = (authorizationHeader: string): string =>
```

Comment:
Maybe this name is better?
```suggestion
export const stripBearerPrefix = (authorizationHeader: string): string =>
```